src/Sarif/Core/RunAutomationDetails.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// Information that describes a run's identity and role within an engineering system process.
+    /// </summary>
+    public partial class RunAutomationDetails
```

Comment:
>RunAutomationDetails [](http://example.com/codeflow?start=25&length=20)

where are the other checks for this type?

Reply:
thanks for the suggestion.
I'm adding the remaining logic :)